pkg/controller/vpc_nat_gateway.go

```diff
@@ -320,14 +320,19 @@ func (c *Controller) handleInitVpcNatGw(key string) error {
 		return fmt.Errorf("failed to get subnet %s", gw.Spec.Subnet)
 	}
 
+	if err := c.updateCrdNatGw(gw.Name); err != nil {
```

Comment:
Why we need update here?

Reply:
if user create a new vpc-nat-gw，need to update vpc and subnet label into vpc-nat-gw, so that pod_iptables_eip.go can find vpc by these two label to get only one vpc-gw-gw pod to create eip,  updateCrdNatGw in init.go  update these two labels for old vpc-nat-gw, this line for new vpc-nat-gw.